src/services/calendar/index.js

```diff
@@ -265,7 +267,7 @@ class Service {
 		};
 
 		return request(options).then((events) => {
-			events = (events.data || events || []);
+			events = events.data || events || [];
```

Comment:
Codacy found an issue: [Assignment to function parameter 'events'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)